src/commands/link/index.ts

```diff
@@ -109,6 +111,38 @@ export default class LinkIndex extends Command {
       throw new Error(chalk.red("No .git found in this directory. Please initialize a git repository with `git init`."));
     }
 
+    // Check if the current branch is 'main'
+    let currentBranch = "";
+    try {
+      currentBranch = execSync("git symbolic-ref --short HEAD", { encoding: "utf-8" }).trim();
+    } catch (error) {
+      this.log(chalk.red("Unable to determine the current branch. Are you inside a Git repository?"));
```

Comment:
asking the user a question is strange, we know theyre in a git repository or else the earlier gitConfigFilePath wouldve failed. we can leave it at unable to determine the current branch.